server/views/pages/induction/checkYourAnswers/partials/_educationAndTraining.njk

```diff
@@ -2,10 +2,14 @@
 
 {% if inductionDto.previousQualifications.qualifications.length %}
 
-  <h3 class="govuk-heading-s govuk-!-margin-top-2 govuk-!-margin-bottom-3">
-    Educational qualifications
-    <span><a class="gov-link govuk-!-font-weight-regular" style="float:right; color:#1d70b8" href="/prisoners/{{ prisonerSummary.prisonNumber }}/induction/qualifications" data-qa="qualificationsLink">Change <span class="govuk-visually-hidden"> there educational qualifications</span></a></span>
```

Comment:
I found another example of some nasty inline styling that we've inherited from the CIAG UI, so have corrected it in the same way that I did the other example the other week